samples/pmic/native/npm2100_one_button/src/main.c

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/logging/log_ctrl.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/mfd/npm2100.h>
+#include <zephyr/drivers/regulator.h>
+
+#define FAST_FLASH_MS	100
```

Comment:
nitpick: naming of these defines could follow the same pattern. E.g.  `FLASH_FAST_MS` instead of `FAST_FLASH_MS`